botocore/serialize.py

```diff
@@ -453,6 +455,11 @@ def _serialize_payload(self, partitioned, parameters,
             serialized['body'] = self._serialize_body_params(
                 partitioned['body_kwargs'], shape)
 
+    def _encode_payload(self, body):
+        if isinstance(body, six.text_type):
+            return body.encode(self.DEFAULT_ENCODING)
```

Comment:
Out of curiosity, is the reason you used ``DEFAULT_ENCODING`` is because it is in a base class? It seems that in the more specific serializers this variable (whose value is ``utf-8``) is not used and straight ``utf-8`` is used.